src/m365/flow/commands/run/run-list.ts

```diff
@@ -151,6 +161,23 @@ class FlowRunListCommand extends PowerAutomateCommand {
     }
     return filters;
   }
+
+  private async retrieveTriggerInformation(items: PowerAutomateFlowRun[]): Promise<void> {
+    const tasks = items.map(async (item: any) => {
+      const requestOptions: CliRequestOptions = {
+        url: item.properties.trigger.outputsLink.uri,
+        headers: {
+          accept: 'application/json',
+          'x-anonymous': true
+        },
+        responseType: 'json'
+      };
+      const response = await request.get<{ body: any }>(requestOptions);
```

Comment:
could we add a type for that to avoid `any`?

Reply:
This is quite hard, as for example, with a manual trigger, this would be something like this:

```json
{
      "text": "test",
      "boolean": false,
      "number": 10
}
```

But for a flow using a more complex trigger, for example a SharePoint trigger, it will be something like this:

```json
{
      "@odata.etag": "\"1\"",
      "ItemInternalId": "5606",
      "ID": 5606,
      "Title": "AAAAA",
      "OData__x0063_": [],
      "OData__x0063_@odata.type": "#Collection(Microsoft.Azure.Connectors.SharePoint.SPListExpandedReference)",
      "OData__x0063_#Id": [],
      "OData__x0063_#Id@odata.type": "#Collection(Int64)",
      "Modified": "2024-05-02T16:08:21Z",
      "Created": "2024-05-02T16:08:21Z",
      "Author": {
        "@odata.type": "#Microsoft.Azure.Connectors.SharePoint.SPListExpandedUser",
        "Claims": "i:0#.f|membership|mathijs@mathijsdev2.onmicrosoft.com",
        "DisplayName": "Mathijs Verbeeck",
        "Email": "mathijs@mathijsdev2.onmicrosoft.com",
        "Picture": "https://mathijsdev2.sharepoint.com/_layouts/15/UserPhoto.aspx?Size=L&AccountName=mathijs@mathijsdev2.onmicrosoft.com",
        "Department": null,
        "JobTitle": "Test update"
      },
      "Author#Claims": "i:0#.f|membership|mathijs@mathijsdev2.onmicrosoft.com",
      "Editor": {
        "@odata.type": "#Microsoft.Azure.Connectors.SharePoint.SPListExpandedUser",
        "Claims": "i:0#.f|membership|mathijs@mathijsdev2.onmicrosoft.com",
        "DisplayName": "Mathijs Verbeeck",
        "Email": "mathijs@mathijsdev2.onmicrosoft.com",
        "Picture": "https://mathijsdev2.sharepoint.com/_layouts/15/UserPhoto.aspx?Size=L&AccountName=mathijs@mathijsdev2.onmicrosoft.com",
        "Department": null,
        "JobTitle": "Test update"
      },
      "Editor#Claims": "i:0#.f|membership|mathijs@mathijsdev2.onmicrosoft.com",
      "{Identifier}": "Lists%252faaaaaa%252f5606_.000",
      "{IsFolder}": false,
      "{Thumbnail}": {
        "Large": null,
        "Medium": null,
        "Small": null
      },
      "{Link}": "https://mathijsdev2.sharepoint.com/_layouts/15/listform.aspx?PageType=4&ListId=b9066cf6%2Df8cd%2D44cd%2Da4a7%2D12eb90c51073&ID=5606&ContentTypeID=0x010057A828478E855B40B7F5D3ED04B07718006EC9CE2C4AFE6C45BCB6015E666DC875",
      "{Name}": "AAAAA",
      "{FilenameWithExtension}": "AAAAA",
      "{Path}": "Lists/aaaaaa/",
      "{FullPath}": "Lists/aaaaaa/5606_.000",
      "{ContentType}": {
        "@odata.type": "#Microsoft.Azure.Connectors.SharePoint.SPListExpandedContentType",
        "Id": "0x010057A828478E855B40B7F5D3ED04B07718006EC9CE2C4AFE6C45BCB6015E666DC875",
        "Name": "Item"
      },
      "{ContentType}#Id": "0x010057A828478E855B40B7F5D3ED04B07718006EC9CE2C4AFE6C45BCB6015E666DC875",
      "{HasAttachments}": false,
      "{VersionNumber}": "1.0",
      "{TriggerWindowStartToken}": "MTszO2I5MDY2Y2Y2LWY4Y2QtNDRjZC1hNGE3LTEyZWI5MGM1MTA3Mzs2Mzg1MDI2Mjg4MTYwMzAwMDA7NzE3MzkxMTcx",
      "{TriggerWindowEndToken}": "MTszO2I5MDY2Y2Y2LWY4Y2QtNDRjZC1hNGE3LTEyZWI5MGM1MTA3Mzs2Mzg1MDI2MjkzMjgxMzAwMDA7NzE3MzkxMjIy"
}
```

Reply:
Indeed tricky. Ok good it. Let's leave it as is then 👍